plugins/catalog/src/components/CatalogPage/CatalogLayout.tsx

```diff
@@ -38,9 +44,14 @@ const CatalogLayout = ({ children }: Props) => {
   return (
     <Page themeId="home">
       <Header
-        title={`${greeting.greeting}, ${profile.displayName || userId}!`}
-        subtitle={`${orgName || 'Backstage'} Service Catalog`}
-        tooltip={greeting.language}
+        title={
+          customHeaderTitle ??
+          `${greeting.greeting}, ${profile.displayName || userId}!`
```

Comment:
We're actually happy to rip out this existing greeting message, as it is really intended for a home page, which the catalog index page is not. If that's good enough of a solution for you we could just change it something very generic like "Software Catalog", or just `${config.getOptionalString('app.title') ?? 'Backstage'} Catalog`.

Reply:
Your suggestion `${config.getOptionalString('app.title') ?? 'Backstage'} Catalog` would be nice. That would be done by you guys in some near future? So should we abandon this PR?

Reply:
We'd create an issue for it at least to give people from the community the opportunity to pick it up. Beyond that we'd have to weight the priority with other work, and I'm not sure were that stands atm.

Reply:
There was issue #3529 but its closed one. Will you create a new one or re-activate the before mentioned?